mlflow/fastai.py

```diff
@@ -380,7 +380,9 @@ def load_model(model_uri):
 
 @experimental
 @autologging_integration(FLAVOR_NAME)
-def autolog(log_models=True, disable=False, exclusive=False):  # pylint: disable=unused-argument
+def autolog(
```

Comment:
Looks like we're missing `disable_for_unsupported_versions` in `mlflow.autolog()`. Can we add this argument to `mlflow.autolog()` (defined in `mlflow/fluent.py`) & ensure that we have test coverage in https://github.com/mlflow/mlflow/blob/master/tests/tracking/fluent/test_fluent_autolog.py#L123-L137?